src/components/modals/Modal.vue

```diff
@@ -77,6 +93,34 @@ export default {
       color: red;
     }
   }
+  > .maximize-toggle {
+    z-index: 100;
+    position: absolute;
+    right: 52px;
+    top: 20px;
+    cursor: pointer;
+    &:hover {
+      color: red;
+    }
+  }
+}
+
+.not-maximized {
+  max-height: 80%;
+  max-width: 80%;
+}
```

Comment:
Ok - I assumed you wouldn't allow this because doing so overwrites the max-width:60% that previously existed in .modal (and which applied to all modals except characters/night-reference/votehistory/).

Reply:
That's fine for now, we can still make the other modals smaller individually later, if they're too big.